x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/recovery_list/recovery_list.tsx

```diff
@@ -0,0 +1,171 @@
+      <SectionError
+        title={
+          <FormattedMessage
+            id="xpack.snapshotRestore.recoveryList.LoadingRecoveriesErrorMessage"
+            defaultMessage="Error loading recoveries"
+          />
+        }
+        error={error}
+      />
+    );
+  } else if (recoveries && recoveries.length === 0) {
+    content = (
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        title={
+          <h1>
+            <FormattedMessage
+              id="xpack.snapshotRestore.recoveryList.emptyPromptTitle"
+              defaultMessage="You don't have any snapshot recoveries"
+            />
+          </h1>
+        }
+        body={
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.snapshotRestore.recoveryList.emptyPromptDescription"
+                defaultMessage="Track progress of indices that are recovered from snapshots."
+              />
+            </p>
+          </Fragment>
+        }
+        data-test-subj="emptyPrompt"
+      />
+    );
+  } else {
+    content = (
+      <Fragment>
+        <EuiFlexGroup alignItems="center" justifyContent="flexStart" gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiPopover
+              id="srRecoveryListIntervalMenu"
+              button={
+                <EuiButtonEmpty
+                  size="xs"
+                  type="text"
+                  iconType="arrowDown"
+                  iconSide="right"
+                  onClick={() => setIsIntervalMenuOpen(!isIntervalMenuOpen)}
+                >
+                  <FormattedMessage
+                    id="xpack.snapshotRestore.recoveryList.intervalMenuButtonText"
+                    defaultMessage="Refresh data every: {interval}"
+                    values={{
+                      interval: (
+                        <FormattedMessage
+                          id="xpack.snapshotRestore.recoveryList.intervalMenu.intervalValue"
+                          defaultMessage="{seconds} {seconds, plural, one {second} other {seconds}}"
+                          values={{ seconds: Math.ceil(currentInterval / 1000) }}
+                        />
+                      ),
+                    }}
+                  />
+                </EuiButtonEmpty>
+              }
+              isOpen={isIntervalMenuOpen}
+              closePopover={() => setIsIntervalMenuOpen(false)}
+              panelPaddingSize="none"
+              anchorPosition="downLeft"
+            >
+              <EuiContextMenuPanel
+                items={INTERVAL_OPTIONS.map(interval => (
+                  <EuiContextMenuItem
+                    key={interval}
+                    icon="empty"
+                    onClick={() => {
+                      changeInterval(interval);
+                      setCurrentInterval(interval);
+                      setIsIntervalMenuOpen(false);
+                    }}
+                  >
+                    {interval >= 60 * 1000 ? (
```

Comment:
if you follow @cjcenizal suggestion, maybe you can also update this to use the `const`?